mri/parallel_mri/extract_sensitivity_maps.py

```diff
@@ -93,6 +94,43 @@ def gridded_inverse_fourier_transform_nd(kspace_loc,
         pfft.ifftn(pfft.ifftshift(gridded_kspace))), 1, 0)
 
 
+def gridded_inverse_fourier_transform_stack(plane_samples, z_samples,
+                                            stack_len, kspace_data, grid,
+                                            method):
+    """
+    This function calculates the gridded Inverse fourier transform
+    from Interpolated non-Cartesian data into a cartesian grid. However,
+    the IFFT is done similar to Stacked FOurier transform.
+
+    Parameters
+    ----------
+    kspace_loc: np.ndarray
```

Comment:
the doc doesn't correspond to the function parameters (I think `kspace_loc` is beter than `plane_samples`, it gives a better indication of what this variable is, i.e. positions)